ansible/host_vars/README.md

```diff
@@ -16,6 +16,8 @@ You will always have to set the following variables to configure a host:
 Variables that _might_ be available for you to change depending on
 what init system your host will be running:
 
+- `is_benchmark`: set to `true`/`false`. If true, will run the
```

Comment:
I'm not sure this would be the way that @jbergstroem.  Instead I think I would have expected a new role that inherits from jenkins worker but does additional configuration.  I could be wrong though.  @jbergstroem can you comment.

Reply:
Looking at it in more details I think I better understand the issue, the naming of our machines still uses test_-need something to identify what role.  We might want to rename the benchmarking machines to start with bench- instead  and then use that to select the role.